src/main/scala/ing/wbaa/druid/DruidClient.scala

```diff
@@ -69,9 +71,17 @@ object DruidClient extends FailFastCirceSupport with JavaTimeDecoders {
     }
   }
 
+  private def handleResponseAsStream(
+      response: HttpResponse
+  )(implicit druidConfig: DruidConfig): Source[DruidResult, Any] =
```

Comment:
What is this `Any`? Should it also become `NotUsed`?

Reply:
`akka.http.scaladsl.model.HttpEntity#dataBytes` returns `Source[ByteString, Any]`. After a short API review of Akka Streams, we can use the `mapMaterializedValue` function of `akka.stream.scaladsl.Source` in order to map `Any` to `NotUsed`. I'll fix it.